packages/common/src/utils/internal/arcgisToGeoJson.ts

```diff
@@ -0,0 +1,249 @@
+// @terraformer/arcgis
+// https://github.com/terraformer-js/terraformer/blob/main/packages/arcgis/README.md
```

Comment:
I'm curious why we need to copy this into the project? I suspect either we can't import it for some reason, or it imports way more than we need?

Reply:
agree, according to bundlephobia `@terraformer/arcgis` is tree-shakable, so we _should_ only end up w/ what we need: https://bundlephobia.com/package/@terraformer/arcgis@2.1.2